gruntfile.js

```diff
@@ -102,6 +102,12 @@ module.exports = function(grunt) {
                 }
             }
         },
+        download: {
+            foo: {
+                src: ['https://dl.openfin.co/services/download?fileName=OpenFinD3FC&config=http://localhost:5000/app.json'],
```

Comment:
I don't think we want a 'foo'.

Do you think it's worth parameterising the port number and project name too?